package.json

```diff
@@ -9,6 +9,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-map-gl": "^5.2.2",
+    "react-router-dom": "^5.1.2",
```

Comment:
We don't seem to be using react router. Is there a reason you have imported it?

Reply:
Oh i was going to use it for the links but end up using <a href>